[  107s] E: cloud-init: init.d-script-missing-dependency-on-remote_fs etc/init.d/cloud-final: required-stop
[  107s] N: 
[  107s] N:    The given init script seems to refer to /usr, possibly using a file or
[  107s] N:    binary from there. Without a dependency on $remote_fs in Required-Start
[  107s] N:    or Required-Stop, as appropriate, the init script might be run before
[  107s] N:    /usr is mounted or after it's unmounted.
[  107s] N:    
[  107s] N:    Using Should-Start or Should-Stop to declare the dependency is
[  107s] N:    conceptually incorrect since the $remote_fs facility is always
[  107s] N:    available. Required-Start or Required-Stop should be used instead. Also,
[  107s] N:    please note that $all should not be used in Required-Stop, only
[  107s] N:    Required-Start.
[  107s] N:    
[  107s] N:    Refer to https://wiki.debian.org/LSBInitScripts for details.
[  107s] N:    
[  107s] N:    Severity: important, Certainty: possible
[  107s] N:    
[  107s] N:    Check: init.d, Type: binary
[  107s] N: 
[  107s] E: cloud-init: init.d-script-depends-on-all-virtual-facility etc/init.d/cloud-final required-start
[  107s] N: 
[  107s] N:    The given init script declares a dependency on the virtual facility
[  107s] N:    "$all".
[  107s] N:    
[  107s] N:    This virtual facility is reserved for local scripts.
[  107s] N:    
[  107s] N:    Moreover, using $all in more than one init.d script is totally broken.
[  107s] N:    
[  107s] N:    Refer to https://wiki.debian.org/LSBInitScripts for details.
[  107s] N:    
[  107s] N:    Severity: serious, Certainty: possible
[  107s] N:    
[  107s] N:    Check: init.d, Type: binary
[  107s] N: 
[  107s] E: cloud-init: init.d-script-missing-dependency-on-remote_fs etc/init.d/cloud-config: required-start
[  107s] E: cloud-init: init.d-script-missing-dependency-on-remote_fs etc/init.d/cloud-config: required-stop
[  107s] E: cloud-init: init.d-script-missing-dependency-on-remote_fs etc/init.d/cloud-init-local: required-stop

--- a/sysvinit/debian/cloud-final
+++ b/sysvinit/debian/cloud-final
@@ -1,7 +1,7 @@
 #! /bin/sh
 ### BEGIN INIT INFO
 # Provides:          cloud-final
-# Required-Start:    $all cloud-config
+# Required-Start:    $remote_fs cloud-config
 # Required-Stop:     $remote_fs
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
